### ISSUE TITLE
Which commit corresponds to the PyPi 0.2.1 release?

### ISSUE THREAD

**Issue by boldorider4** [facebookresearch/fairseq2#769]
At owner:AnacondaRecipes, we are trying to build `fairseq2 0.2.1`, but noticed the GH project is not tagged at that version, but looking at the history, [this commit](https://github.com/facebookresearch/fairseq2/commit/c664a42ac30c5ba3ab23382327903cf690ea1ffb) seems to correspond to the PyPi release. Can you confirm?

**Comment by zyaoj** [comment]
@boldorider4 
Yes this is the PyPi release for [v0.2.1](https://github.com/facebookresearch/fairseq2/tree/refs/heads/v0.2.1)
